bmi_lstm.py

```diff
@@ -580,12 +580,12 @@ def get_value_at_indices(self, var_name, dest, indices):
         array_like
             Values at indices.
         """
-        #JMFrame: chances are that the index will be zero, so let's include that logic
-        if np.array(self.get_value(var_name)).flatten().shape[0] == 1:
-            return self.get_value(var_name)
-        else:
-            val_array = self.get_value(var_name).flatten()
-            return np.array([val_array[i] for i in indices])
+        #NJF This must copy into dest!!!
+        tmp = self.get_value_ptr(var_name)
+        #Convert to np.array in case of singleton/non numpy type, then flatten
+        data = np.array(self.get_value(var_name)).flatten()
```

Comment:
I think you meant to use the `tmp` variable you created, but didn't.

Once that is changed, do we need support for non-numpy types for the BMI functions?  I'm not sure how we could implement `get_value_ptr` properly in Python without it using a numpy array.

Reply:
`data = np.array(self.get_value(var_name)).flatten()` the type internally can be anything that `np.array` can convert....worse case scenario you get a numpy byte array....so either way I think this works....just have to have the numpy interface for the `ptr` part to work correctly with the numpy memory views.

Reply:
@robertbartel I'm going to close this PR since the same fix is included in #23.  I did remove the tmp variable, that was a debug variable to crept in.